Changes

```diff
@@ -1,3 +1,12 @@
+0.06 2018-10-??
+ - GCE receives a new attribute "region" to select the region where the registry
+   is hosted.
+ - New Azure Container Registry provider (ACR). Can authenticate with admin credentials
+   or Azure AD credentials.
+ - Internal refactor
+   - using Moo and Types::Standard instead of Moose
+   - pass the scope to the singnature methdo in auth modules
```

Comment:
There are some typo's in this sentence: `signature method`

Reply:
"tahnk oyu" 🤣